src/components/PetsList.jsx

```diff
@@ -1,19 +1,87 @@
 import FilterList from "./FilterList";
 import PetCard from "./PetCard";
 import { PetItems } from "../files/PetsData";
-import React from "react";
+import React, { useState } from "react";
+import CategoriesFilterCollapse from "./CategoriesFilterCollapse";
 
 const PetsList = () => {
+  const [filteredItems, setFilteredItems] = useState([]);
```

Comment:
I think naming this `filteredPets` would make this stand out more. Right now we have `filteredItems` and `filterItems` which are very close to the same name.